packages/core/src/mm/actors/Player.S

```diff
@@ -1224,3 +1224,7817 @@ PATCH_END
 PATCH_START 0x8085e958
 .int 0
 PATCH_END
+
+/* Hammer Related ASM */
+
+PATCH_START 0x80123960
+  addiu     sp, sp, -0x20
+  sw        ra, 0x0014 (sp)
+  jal Player_CustomActionToModelGroup
+  nop
+  lw        ra, 0x0014 (sp)
+  jr        ra
+  addiu     sp, sp, 0x20
+PATCH_END
+
+PATCH_START 0x80128DDC
+  addiu     a0, s2, 0x0
+  jal       Player_GetCustomSwordLength
+  addiu     a1, s1, 0x0
+  j         0x80128E24
```

Comment:
I don't know about the others, but my personal preference for this would be to add `nop` 15 times here since that code becomes unreachable.

Reply:
I think it's fine.